src/App.jsx

```diff
@@ -10,12 +8,26 @@ function App() {
   ];
   const [personsToSendAGift, setPersonsToSendAGift] = useState(initialPersons);
 
+  function handleSubmit(event) {
+    event.preventDefault();
+    const newPerson = event.target.newPerson.value;
+    const newIdeasString = event.target.newIdeas.value;
+    const newIdeasArray = newIdeasString.split(', ');
+    const newPersonObject = { name: newPerson, ideas: newIdeasArray };
+    setPersonsToSendAGift([...personsToSendAGift, newPersonObject]);
+    console.log(personsToSendAGift);
+  }
+
   return (
     <div className="App">
       <h1>Weihnachts-Ideen</h1>
-      {personsToSendAGift.map((person, index) => (
-        <Card key={index} personToBeGifted={person} />
-      ))}
+      <form onSubmit={handleSubmit}>
+        <label htmlFor="newPerson">Add a new person</label>
+        <input type="text" id="newPerson" name="newPerson" />
+        <label htmlFor="newIdeas">Add your ideas</label>
```

Comment:
It would be great if the user gets a note on how to enter the ideas (comma-separated)